tests/ai_safety/evalhub/constants.py

```
@@ -77,6 +79,12 @@
     "seccompProfile": {"type": "RuntimeDefault"},
 }
 
+# Minimal MinIO for simple-mode intents (no DSPA needed)
+SIMPLE_MINIO_ACCESS_KEY: str = "minioadmin"
+SIMPLE_MINIO_SECRET_KEY: str = "minioadmin"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "SIMPLE_MINIO_SECRET_KEY"

(S105)